mixer/README.md

```diff
@@ -1,5 +1,30 @@
 # Mixer
 
+## Deprecation notice
+
+As of Istio 1.5, Mixer is deprecated. The functionality provided by Mixer is
+being moved into the Envoy proxies. Use of Mixer with Istio will only be
+supported through the 1.7 release of Istio.
+
+## Mixer's future
+
+Because of the extensive feedback we received from the community about the
+difficulties encountered with Mixer, we are making a substantial change to
+Istio's architecture. By taking advantage of the work being done in the
+[WebAssembly](https://webassembly.org) community, we are working to move the
+functionality that formerly lived in Mixer into the Envoy proxy itself.
+
+It reduces the complexity of installation (fewer components, fewer CRDs),
+makes it easier to run Istio, and importantly it allows integrations in a more
```

Comment:
This sentence is a little clunky. How is it more standard? I'd suggest bullets as well to break up the sentence.

I am not a mixer wizard - perhaps @nrjpoddar could offer better wording.
```
This change to Istio's infrastructure reduces complexity by:
- There are fewer components to install
- There are fewer CRDs to manage
- Istio is simpler to manage and run
- Integrations are becoming standardized
```

Reply:
could probably remove `by` in my comment as it is unneeded.

Reply:
The "standardization" is because now _all_ integrations with Envoy, whether Istio-related or not, have a standard method of integration (instead of APIs that only work w/ Istio). I'm happy to make this a bulleted list though (and I hope we don't worry _too_ much about these docs. This is not intended to be the Extensions v2 doc, just an explanation of why people shouldn't be doing Mixer integrations anymore!